package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV3.java

```diff
@@ -104,6 +104,7 @@ public void upload ( final File file, final String filename ) throws IOException
         }
         catch ( final IOException e )
         {
+            e.printStackTrace ( this.listener.error ( "Failed to create upload archive" ) );
```

Comment:
@chrismathis  `this.listener` is the jenkins logger to print to the build console so `this.listener.error` is enough